src/doppelkopfSchreiben.js

```diff
@@ -40,12 +63,18 @@ window.onload = () => {
       addPlayers(winnerSelect, players);
       winnerSelect.selectedIndex = -1;
     }
-    for (let i = 0; i < 10; i += 1) {
-      const specialPoints = document.getElementById('special-points');
-      const opt = document.createElement('option');
-      opt.value = i;
-      opt.innerHTML = i;
-      specialPoints.appendChild(opt);
-    }
   });
+
+  for (let i = 0; i < 10; i += 1) {
+    const specialPoints = document.getElementById('special-points');
+    const opt = document.createElement('option');
+    opt.value = i;
+    opt.textContent = i;
```

Comment:
Habe das hier oben aus dem callback rausgezogen, weil es nichts mit der response vom `/scorepads` request zu tun hat. Das hier können wir direkt initialisieren.

Außerdem habe ich `innerHTML` hier ersetzt - ich weiß, dass das in vielen Beispielen im Internet verwendet wird, aber die sind entweder veraltet oder nicht ideal. Meistens willst du eher `textContent`, außer du hast einen wirklich guten Grund `innerHTML` zu verwenden. Mit `innerHTML` kannst du tatsächlich auch HTML schreiben, wie zum Beispiel: `opt.innerHTML = <span>asd</span>`. Theoretisch kann man damit die Seite zerschießen. Hier kann das eher nicht passieren, aber trotzdem ist es fast immer eine gute Idee was anderes als `innerHTML` zu verwenden.